src/SourceBuild/tarball/content/repos/Directory.Build.targets

```diff
@@ -219,9 +233,14 @@
         '$(VSS_NUGET_EXTERNAL_FEED_ENDPOINTS)' != '' and
         '$(SetUpInternalTransportFeed)' == 'true'"
       NuGetConfigFile="%(NuGetConfigFiles.Identity)"
-      SourceName="dotnet5-internal-transport"
+      SourceName="$(DotNet5InternalTransportNuGetSourceName)"
       SourcePath="https://pkgs.dev.azure.com/dnceng/internal/_packaging/dotnet5-internal-transport/nuget/v3/index.json" />
 
+    <UpdateNuGetConfigPackageSourcesMappings
```

Comment:
Can you explain how `UpdateNuGetConfigPackageSourcesMappings` works in conjunction with `RemoveInternetSourcesFromNuGetConfig` in the context of an online build? Is my understanding correct that `RemoveInternetSourcesFromNuGetConfig` only removes `dnceng` package sources when running an online build?

And in an offline build, we remove the all package source mappings and replace them with our own?

Reply:
> Can you explain how UpdateNuGetConfigPackageSourcesMappings works in conjunction with RemoveInternetSourcesFromNuGetConfig in the context of an online build?

The intent is for them to be independent.  That being said, I think RemoveInternetSourcesFromNuGetConfig needs to be updated to remove any package source mappings that are associated with removed sources.

> And in an offline build, we remove the all package source mappings and replace them with our own?

Correct

Reply:
I think I will log a follow-up issue to fix this so that I can unblock others.